package.json

```diff
@@ -214,7 +214,7 @@
     "@scaleway/random-name": "2.1.1",
     "@storybook/react": "6.3.12",
     "@types/intl-tel-input": "17.0.4",
-    "@types/react-datepicker": "4.3.0",
+    "@types/react-datepicker": "4.3.1",
```

Comment:
intended to be in deps ?

Reply:
@types/react-datepicker is used inside DateInput, so we need it inside our build